clippy_lints/src/large_enum_variant.rs

```diff
@@ -96,11 +97,12 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for LargeEnumVariant {
                                     VariantData::Unit(_) => unreachable!(),
                                 };
                                 if let Some(snip) = snippet_opt(cx, span) {
-                                    db.span_suggestion(
+                                    db.span_suggestion_with_applicability(
                                         span,
                                         "consider boxing the large fields to reduce the total size of the \
                                          enum",
                                         format!("Box<{}>", snip),
+                                        Applicability::Unspecified,
```

Comment:
not MachineApplicable since changing the enum field can stop the code from compiling (since we don't also change all the sites where you destructure the enum and would need to add the box to the pattern matching)

Reply:
`MaybeIncorrect` I believe.